2278.percentage-of-letter-in-string.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/percentage-of-letter-in-string/
+ */
+
+/**
+ * @param {string} s
+ * @param {character} letter
+ * @return {number}
+ */
+var percentageLetter = function(s, letter) {
+    //Picked up same alphabet from s
```

Comment:
This comment may be confusing. 
The comment will make more sense if you write like this.
"Extract all the same alphabets as letter from s"